v3/pkg/application/webview_window_darwin.m

```diff
@@ -241,6 +241,9 @@ - (void)webView:(nonnull WKWebView *)webView stopURLSchemeTask:(nonnull id<WKURL
         }
     }
 }
+- (NSApplicationPresentationOptions)window:(NSWindow *)window willUseFullScreenPresentationOptions:(NSApplicationPresentationOptions)proposedOptions {
```

Comment:
The main change is here: a new property on the delegate interface is used as a flag to either keep the default presentation options, or enable the `NSApplicationPresentationAutoHideToolbar` option, thus hiding the toolbar.